src/functions/asyncApi2PayloadValidation.ts

```diff
@@ -0,0 +1,60 @@
+import * as AJV from 'ajv';
+import * as jsonSpecv7 from 'ajv/lib/refs/json-schema-draft-07.json';
```

Comment:
Same. We need to find some way to avoid doing this.
I think I have an idea on how to expose it. Will open a PR today.

Reply:
@nulltoken https://github.com/stoplightio/spectral/pull/1100

Reply:
@P0lip #1100 is merged and this PR has been rebased on top of it in order to leverage it.

Sadly, I'm still unable to migrate this to a custom function (as rollup cringes about tslib). Proposal is to keep it as a built-in function and migrate it to a custom function once this is merged. 

I've updated the description of this PR to keep track of this through an upcoming issue.